### ISSUE TITLE
Changing a fork doesn't pick up new repo

### ISSUE THREAD

**Issue by airblade** [airblade/voom#1]
1. Declare a repo (which has a fork) in `plugins` and `voom install`.
2. Update declaration in `plugins` to one of the fork(s) and `voom install`.
3. Then
   - Expected: original repo is removed and new repo is installed.
   - Actual: original repo is left and new repo is not installed.

**Comment by airblade** [comment]
Plugins are currently installed under `pack/voom/start/`.  The name `voom` is arbitrary: vim will find anything under `pack/*/start/`.

So instead of using `voom` as the namespace for all plugins, we could use something unique for a particular fork of a plugin.  For example, the GitHub username.  So a plugin declared as `tpope/vim-fugitive` would be installed to `pack/tpope/vim-fugitive` (instead of `pack/voom/vim-fugitive`).

Clearly not all plugins are installed from GitHub.  However whether they are local (`~/code/src/vim-gitgutter`) or online, they all have some kind of path.  We could just use the penultimate segment of the path.

**Comment by eNV25** [comment]
We could also replace any `/` with a replacement character, e.g. with `-` `airblade/voom` would become `airblade-voom`. This would solve the namespacing issue, and there would be no need to stop using the `voom` plugin.

**Comment by airblade** [comment]
Yes, that would work too.  It would also keep all the plugins in the same directory, which doesn't really matter but feels neater.

It would make it slightly harder to spot a particular plugin in that directory, if you wanted to for some reason.  To improve that we could swap the names, e.g. `voom-airblade` or `fugitive-tpope`.  Again, though, it doesn't really matter.
